pyuvsim/data/test_config/param_1time_1src_testvot.yaml

```diff
@@ -7,6 +7,11 @@ freq:
   channel_width: 1000000.0
 sources:
   catalog: '../single_source.vot'
+  table_name: single
```

Comment:
We should probably have it in the documentation that these additional fields are necessary.

Reply:
I updated the docs.